### PR TITLE
Feat: add components

### PR DESCRIPTION
**Pull request by Shailseen**
### Added components
**Components have to check**

- `Lists`: Different styles of list such as **unorder and ordered lists**.

- `Grid`: Different layout of grids has been shown.

- `Toast`: Toast with added functionality with **live demo**.
 
- `Modal`: This is also given with Live demo you can easily use with copy the **HTML** as well as **JS**.
 
image
image
image
image
Please review and give feedback. :)

### CODE REVIEW

File: component css/lists.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+.lists-container ol{
+    flex-grow: 1;
+}
+.list-style-smAlphabet{
+    list-style-type: lower-alpha;
+}
+.list-style-upperAlphabet{
+    list-style-type: upper-alpha;
+}
+.list-style-upperRoman{
```
Comment (vishalg8454) [inline component css/lists.css:23]: avoid camelCase in CSS.

File: doc-css/nav.css [hunk 2]
```diff
@@ -3,7 +3,7 @@
     grid-area: header;
     width: 100%;
     height: 10vh;
-    position: sticky;
+    /* position: sticky; */
```
Comment (vishalg8454) [inline doc-css/nav.css:6]: remove commented code.

File: component css/grid.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
+.grid-first{
+    grid-area: grid-first;
+    background-color: aqua;
+    height: 3rem;
+}
+.grid-second{
+    grid-area: grid-second;
+    height: 3rem;
+    background-color: aquamarine;
```
Comment (vishalg8454) [inline component css/grid.css:9]: it's better if you put non-CL classes into a separate file like `docs.css`

**Review by vishalg8454 — APPROVED** [review]
Good UI, nice component library. Left a few suggestions,. Can merge.

File: component css/lists.css (hunk 1 above)
Comment (Shailseen) [reply component css/lists.css:23]: it just by mistake.

File: doc-css/nav.css (hunk 2 above)
Comment (Shailseen) [reply doc-css/nav.css:6]: sure

File: component css/grid.css (hunk 3 above)
Comment (Shailseen) [reply component css/grid.css:9]: its component library css not for my webpage.

File: component css/grid.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+.aside-rt{
+    height: 10rem;
+    margin-left: 0.5rem;
+    background-color: yellow;
+    grid-area: aside-rt;
+}
+.main-content{
+    height: 10rem;
+    background-color: skyblue;
+    grid-area: main-content;
```
Comment (arjun-computer-geek) [inline component css/grid.css:47]: Use color variable always

**Review by arjun-computer-geek — APPROVED** [review]
Everything looks good.

File: component css/grid.css (hunk 4 above)
Comment (Shailseen) [reply component css/grid.css:47]: yeah. but i will do it later on.

**Pull request merged**